lib/Controller/AccountsController.php

```diff
@@ -346,15 +345,15 @@ public function create(string $accountName, string $emailAddress, string $passwo
 				$account = $this->setup->createNewAccount($accountName, $emailAddress, $imapHost, $imapPort, $imapSslMode, $imapUser, $imapPassword, $smtpHost, $smtpPort, $smtpSslMode, $smtpUser, $smtpPassword, $this->currentUserId);
 			}
 		} catch (CouldNotConnectException $e) {
-			$this->logger->info('Creating account failed: ' . $e->getMessage(), [
-				'exception' => $e,
-			]);
-			return \OCA\Mail\Http\JsonResponse::fail([
+			$data = [
 				'error' => $e->getReason(),
 				'service' => $e->getService(),
 				'host' => $e->getHost(),
 				'port' => $e->getPort(),
-			]);
+			];
+
+			$this->logger->info('Creating account failed: ' . $e->getMessage(), $data);
```

Comment:
Let's hope there will never be a conflict between `$data` and the generated json array :P